Tests/VauxTests/VauxTests.swift

```diff
@@ -293,6 +293,106 @@ final class VauxTests: XCTestCase {
     }
   }
   
+    
+  func testLinkScript() {
+    func masterPage() -> HTML {
+        html {
+            body {
+                linkScript(src: "script.js")
+            }
+        }
+    }
+    let correctHTML = """
+        <!DOCTYPE html>
+        <html>
+          <body>
+            <script src="script.js">
+            </script>
+          </body>
+        </html>
+        """.replacingOccurrences(of: "\n", with: "")
+    let vaux = Vaux()
+    vaux.outputLocation = .file(name: "testing", path: "/tmp/")
+    do {
+        let rendered = try renderForTesting(with: vaux, html: masterPage())
+        XCTAssertEqual(rendered, correctHTML)
+    } catch let error {
+        XCTFail(error.localizedDescription)
+    }
+  }
+    
+  func testScript() {
+    var pageTitle = "Page title"
+    var pageBody = "Page body"
+    func pageWithJavascript() -> HTML {
+        html {
+            body {
+                paragraph { "" }.id("script")
+                script(code: "document.getElementById('script').innerHTML = 'Hello JavaScript!';")
+            }
+        }
+    }
+    let correctHTML = """
+        <!DOCTYPE html>
+        <html>
+          <body>
+            <p id="script">
+            </p>
+            <script>
+              document.getElementById('script').innerHTML = 'Hello JavaScript!';
+            </script>
+          </body>
+        </html>
+        """.replacingOccurrences(of: "\n", with: "")
+    
+    let vaux = Vaux()
+    vaux.outputLocation = .file(name: "testing", path: "/tmp/")
+    do {
+        let rendered = try renderForTesting(with: vaux, html: pageWithJavascript())
+        XCTAssertEqual(rendered, correctHTML)
+    } catch let error {
+        XCTFail(error.localizedDescription)
+    }
+  }
+  
+  func testScriptMultiline() {
+    var pageTitle = "Page title"
+    var pageBody = "Page body"
+    func pageWithJavascript() -> HTML {
+        html {
+            body {
+                paragraph { "" }.id("script")
+                script(code: """
+                    document.getElementById("script").innerHTML = "Hello JavaScript!";
+                    """
+                )
+            }
+        }
+    }
+    let correctHTML = """
+    <!DOCTYPE html>
+    <html>
+      <body>
+        <p id="script">
+        </p>
+        <script>
+          document.getElementById("script").innerHTML = "Hello JavaScript!";
+        </script>
+      </body>
+    </html>
+    """.replacingOccurrences(of: "\n", with: "")
+    
+    let vaux = Vaux()
+    vaux.outputLocation = .file(name: "testing", path: "/tmp/")
+    do {
+        let rendered = try renderForTesting(with: vaux, html: pageWithJavascript())
+        //TODO: fix the HTML encoding of the multiline to make the test pass.
```

Comment:
Any idea on how to fix this one? Using quote in a multiline string escape the quote, so we are not producing what is expected...

Reply:
Yeah, this is going to be a fun one. The way we are running unit tests in the first place is quite daft, so I need some help thinking of an easier way to test this. I also need to think about how to make certain tags write inline rather than with an indent, but that's an easier problem to solve.

Reply:
It was so promising to use multiline, but maybe it is just for the hack of it.
For the tests, I was think about removing all the useless spaces, but the trim method does quite work like that...

Reply:
Yeah, I spent a good couple of hours trying to get string trimming to work, but I think there must be a better way. Either way, I only have one failing test now. Let me know if you want to take a peek at that! I think smaller tests are generally better here.